plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js

```diff
@@ -330,19 +330,34 @@ jQuery( function ( $ ) {
 	);
 
 	// Stock options.
+	function show_or_hide_stock_management_fields(
+		isStockManagementEnabled,
+		productType
+	) {
+		const stockManagementFields = $( '.stock_fields' );
```

Comment:
I wonder if we should convert the `const`s into `var`s. The `const` keyword was introduced to JavaScript with the ECMAScript 2015 specification.

Reply:
I'm pretty sure all the browsers we support support const. Doesn't some of our other compiled code have contacts in the output?

Reply:
https://caniuse.com/const